src/Services/ImpersonateManager.php

```diff
@@ -92,9 +96,12 @@ public function leave()
             $this->app['auth']->quietLogout();
             $this->app['auth']->quietLogin($impersonator);
 
+            $this->extractAuthCookieFromSession();
+
             $this->clear();
 
         } catch (\Exception $e) {
+            dump($e->getMessage());
```

Comment:
Was this `dump()` command supposed to be left in?

Reply:
Nope 👍